src/index.js

```diff
@@ -1,13 +1,14 @@
 import React from 'react'
 import ReactDOM from 'react-dom'
 import './index.css'
-import App from './App'
+// import App from './App'
```

Comment:
podemos excluir esses comentários?

Reply:
esse App é nossa aplicação principal hahahaha ela nao deveria ter sido comentada kkk to tentando fazer com que retorne ao ponto q era antes de fazer o card